### PR TITLE
Replace CallToActionButton with Button, and remove Button

### PR DESCRIPTION
**Pull request by benoneal**

### CODE REVIEW

File: src/components/callstoaction/CallToActionButton/style.scss [hunk 1]
```diff
@@ -1,11 +1,180 @@
...
+  display: inline-block;
+  position: relative;
+  z-index: 1;
+  overflow: hidden;
+  margin: 2px;
+  padding: 0;
+  line-height: 44px;
+  border-width: 2px;
+  border-style: solid;
+  border-radius: 2px;
```
Comment (slwen) [inline src/components/callstoaction/CallToActionButton/style.scss]: Could use `$ehw-border-radius` here.

**Comment by bjbrewster** [comment]
Looks good to me. @slwen ?

File: src/components/callstoaction/CallToActionButton/__tests__/CallToActionButton-test.js [hunk 2]
```diff
@@ -1,22 +1,148 @@
...
+      var onClick = function(){
+        clicked = true;
+      };
+
+      component = TestUtils.renderIntoDocument(
+        <CallToActionButton kind='cta' label='Get Started' icon='chevron-right' onClick={ onClick }/>
+      );
+    });
+
+    it('renders a CTA CallToActionButton', function(){
```
Comment (slwen) [inline src/components/callstoaction/CallToActionButton/__tests__/CallToActionButton-test.js]: Nitpick but "it renders a call to action call to action button" is odd (assuming just remnant from renaming component)

Comment (benoneal) [inline src/components/callstoaction/CallToActionButton/__tests__/CallToActionButton-test.js]: it's a CallToActionButton component, of kind "cta". it's not the cleanest phrasing but it's just a test and logically makes sense. :/

**Pull request merged**